### PR TITLE
Update friends-house

### PR DESCRIPTION
**Pull request by EwyBoy**
Updated:
- Shortened plugin description for better visibility in the Plugin Hub browser window.
- Updated readme.md with an improved plugin description and added images for better showcasing.
- Implemented a build pipeline for automatic tests.
- Added security.md, contributing.md, and code_of_conduct.md to the repository.
- Included issue and pull request templates.
- ~~Updated Lombok and JUnit dependencies.~~

TLDR: Updated GitHub repository stuff and some descriptions, no code changes.

### CODE REVIEW

**Comment by EwyBoy** [comment]
I guess it did not like me updating my dependencies on my end

**Comment by EwyBoy** [comment]
Sorry kinda new to this RuneLite plugin development thing, learning the process and limitations. 
That fixes the build ✅

**Review by iProdigy — APPROVED** [review]

**Review by pajlada — APPROVED** [review]
